src/features/device-state/routes/state-get-v2.ts

```diff
@@ -82,15 +84,15 @@ function buildAppFromRelease(
 		}
 	}
 
-	for (const ipr of release.contains__image as AnyObject[]) {
+	for (const ipr of release.contains__image) {
 		// extract the per-image information
 		const image = ipr.image[0];
 
 		const si = serviceInstallFromImage(device, image);
 		if (si == null) {
 			throw new Error(
 				`Could not find service install for device: '${
-					application.uuid
+					application.id
```

Comment:
This should be the only (tiny) runtime change: It is a mistake captured by the typings, the `uuid` from application was not previously `$select`ed